### PR TITLE
Cleanup Imports

### PR DESCRIPTION
**Pull request by Cabalist**
# Summary

* This is a…
    * [ ] Bug fix
    * [ ] Feature addition
    * [x] Refactoring
    * [ ] Minor / simple change (like a typo)
    * [ ] Other
* **Describe this change in 1-2 sentences**:
Used an automated tool to reorder the imports to be in line with PEP8 and remove unused ones.   

# Problem

I noticed some files had unused imports.   This should not affect functionality at all (the tests pass) and helps the code be better in line with PEP8.

# Solution

I ran an automated tool on the code and then reviewed each file.

# Action

### CODE REVIEW

File: picard/tagger.py [hunk 1]
```diff
@@ -44,44 +42,34 @@ def _patched_shutil_copystat(src, dst, *, follow_symlinks=True):
 _orig_shutil_copystat = shutil.copystat
 shutil.copystat = _patched_shutil_copystat
 
-import picard.resources
```
Comment (samj1912) [inline picard/tagger.py]: This is a required import.

**Comment by Cabalist** [comment]
Noted.   I've updated the PR.

File: picard/tagger.py [hunk 2]
```diff
@@ -44,44 +42,35 @@ def _patched_shutil_copystat(src, dst, *, follow_symlinks=True):
...
 from picard.disc import Disc
 from picard.file import File
 from picard.formats import open_ as open_file
-from picard.track import Track, NonAlbumTrack
+from picard.i18n import setup_gettext
+from picard.plugin import PluginManager
 from picard.releasegroup import ReleaseGroup
-from picard.collection import load_user_collections
-from picard.ui.mainwindow import MainWindow
+import picard.resources  # Required. Do not remove
```
Comment (samj1912) [inline picard/tagger.py]: Wrong place.

File: picard/coverart/image.py [hunk 3]
```diff
@@ -25,24 +25,19 @@
...
 from picard import config, log
 from picard.coverart.utils import translate_caa_type
 from picard.script import ScriptParser
-from picard.util import (
-    decode_filename,
-    encode_filename,
-    replace_win32_incompat,
-    imageinfo
-)
+from picard.util import (decode_filename, encode_filename, imageinfo, replace_win32_incompat)
```
Comment (zas) [inline picard/coverart/image.py]: Ok for the sort, but imho, multiline imports should be kept in the following form, one element per line, with extra comma at the end:

```python
from picard.util import (
    decode_filename,
    encode_filename,
    imageinfo,
    replace_win32_incompat,
)
```

Reasons:

- easier to sort in editors (ie. using `!sort` in vim)
- easier to insert/delete an element (keeping elements ordered, line deletion)
- easier to split when imported stuff is moved in another file
- patches easier to review (one change per line)
- shorter lines
- simpler expressions to search/replace

File: picard/const/__init__.py [hunk 4]
```diff
@@ -17,14 +17,14 @@
...
+import builtins
 import os
-import sys
 
 from PyQt5.QtCore import QStandardPaths
+
 from picard import PICARD_APP_NAME
 
-# Install gettext "noop" function in case const.py gets imported directly.
-import builtins
```
Comment (zas) [inline picard/const/__init__.py]: Even though moving it a top respects overall style, in this case, it was making more sense for it to be near its use, that's `builtins` dict overriding, not a usual import.

File: picard/ui/options/dialog.py [hunk 5]
```diff
@@ -18,31 +18,15 @@
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
 
 from PyQt5 import QtCore, QtWidgets
+
 from picard import config
-from picard.util import webbrowser2, restore_method
-from picard.ui import PicardDialog, HashableTreeWidgetItem
-from picard.ui.util import StandardButton
+from picard.ui import HashableTreeWidgetItem, PicardDialog
 from picard.ui.options import (
```
Comment (zas) [inline picard/ui/options/dialog.py]: Arent't those required ??

File: picard/ui/options/plugins.py [hunk 6]
```diff
@@ -24,16 +24,16 @@
...
 from picard import config, log
-from picard.const import (
-    USER_PLUGIN_DIR,
-    PLUGINS_API,
-    PLUGIN_ACTION_NONE,
-    PLUGIN_ACTION_INSTALL,
-    PLUGIN_ACTION_UPDATE,
-    PLUGIN_ACTION_UNINSTALL,
-)
+from picard.const import (PLUGINS_API,
```
Comment (zas) [inline picard/ui/options/plugins.py]: Ok for re-ordering, but i was preferring the previous style.

**Comment by zas** [comment]
Obsoleted by https://github.com/metabrainz/picard/pull/923 (merged)

**Pull request closed without merging**